crates/solidity/testing/snapshots/cst_output/Identifier/returns/generated/0.4.11.snap

```diff
@@ -0,0 +1,11 @@
+# This file is generated automatically by infrastructure scripts. Please don't edit by hand.
+
+Error: Expected end of input.
```

Comment:
Might be missing something, but shouldn't this error message say that we expected an identifier? The `Expected end of input` one tells me that an identifier was parsed successfully, with additional "unmatched" characters at the end, which is not true, given that the error is at the first character.

Reply:
You are right, I missed this.

Reply:
This will have to wait. It requires that we fork chumsky to get access to the stream offset so that we can properly implement a difference combinator, rather than the current hack.

Reply:
I have forked chumsky, which solves the issue with the performance of the difference operation, but error message æsthetics are a bigger topic than this PR.

Reply:
Thanks for explaining. Sounds good 👍